addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -4229,6 +4229,75 @@ QUnit.test('add custom filter should filter messages accordingly to selected fil
     assert.verifySteps(['message_fetch:id=0,needaction=true'], "A message_fetch request should have been done for selected filter & domain of current thread (inbox)");
 });
 
+QUnit.test("highlight the message mentioning the current user inside the channel", async function (assert) {
```

Comment:
These tests should be thread view tests actually, not specific to discuss. This also works in chat window for example.

Reply:
> These tests should be thread view tests actually, not specific to discuss. This also works in chat window for example.

@seb-odoo Done!! You may check now..

Reply:
Thank you, all good.